src/Compilers/VisualBasic/Test/Semantic/Semantics/UserDefinedBinaryOperators.vb

```diff
@@ -2546,6 +2546,35 @@ End Module
 
         End Sub
 
+
+        <WorkItem(14688, "https://github.com/dotnet/roslyn/issues/14688")>
+        <Fact>
+        Public Sub OperatorsWithDefaultValuesAreNotBound()
+
+            Dim compilationDef =
+<compilation name="OperatorsWithDefaultValuesAreNotBound">
+    <file name="a.vb"><![CDATA[
+Public Class Foo
+    Public Shared Operator Not(Optional x As Foo = Nothing) As Integer 'BIND1:"Foo"
+        Return 0
+    End Operator
+End Class
+    ]]></file>
+</compilation>
+
+            Dim compilation = CreateCompilationWithMscorlibAndVBRuntime(compilationDef)
+            CompilationUtils.AssertTheseDiagnostics(compilation,
+<expected>
+BC33010: 'operator' parameters cannot be declared 'Optional'.
+    Public Shared Operator Not(Optional x As Foo = Nothing) As Integer 'BIND1:"Foo"
+                               ~~~~~~~~
+</expected>)
+
+            Dim model = GetSemanticModel(compilation, "a.vb")
+            Dim node As ExpressionSyntax = CompilationUtils.FindBindingText(Of IdentifierNameSyntax)(compilation, "a.vb", 1)
+            Assert.NotNull(model.GetSymbolInfo(node)) 'Compiler API can retreive information about symbol without exceptions
```

Comment:
Probably good to validate at least that you get a good symbol back.  I.e. it's a NamedTypeSymbol, it's not an ErrorType, and the name is 'Foo'.